eos/utils/integrate-cubature.hh

```diff
@@ -0,0 +1,123 @@
+/* Adaptive multidimensional integration of a vector of integrands.
+ *
+ * Copyright (c) 2005-2013 Steven G. Johnson
+ *
+ * Portions (see comments) based on HIntLib (also distributed under
+ * the GNU GPL, v2 or later), copyright (c) 2002-2005 Rudolf Schuerer.
+ *     (http://www.cosy.sbg.ac.at/~rschuer/hintlib/)
+ *
+ * Portions (see comments) based on GNU GSL (also distributed under
+ * the GNU GPL, v2 or later), copyright (c) 1996-2000 Brian Gough.
+ *     (http://www.gnu.org/software/gsl/)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#ifndef CUBATURE_H
+#define CUBATURE_H
+
+#include <stdlib.h> /* for size_t */
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif /* __cplusplus */
```

Comment:
Since we do not and will not use C in the EOS source code, this (and the corresponding closing brace)
is dead weight.

Reply:
Ok. It was just copied over